specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -0,0 +1,1949 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The fully qualified ID of the managed application, including the managed application name and the managed application resource type. Use the format, /subscriptions/{guid}/resourceGroups/{resource-group-name}/Microsoft.Solutions/applications/{application-name}",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/Application"
+            },
+            "description": "Parameters supplied to update an existing managed application."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Created - Returns information about the managed application, including provisioning status.",
+            "schema": {
+              "$ref": "#/definitions/Application"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Solutions/applications/{applicationName}/updateAccess": {
+      "post": {
+        "tags": [
+          "updateAccess"
+        ],
+        "operationId": "updateAccess_Post",
+        "description": "Update Access on application.",
+        "x-ms-examples": {
+          "Update Jit Access": {
+            "$ref": "./examples/updateJitAccess.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "pattern": "^[-\\w\\._\\(\\)]+$",
+            "minLength": 1,
+            "maxLength": 90
+          },
+          {
+            "name": "applicationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the managed application.",
+            "minLength": 3,
+            "maxLength": 64
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JitUpdateAccessDefinition"
+            },
+            "description": "Parameters supplied to the update managed application access."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Async actions should return 202: 
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#call-action-post

Reply:
So in our code base, it's currently HttpStatusCode.Created(201). So which side should we change first? Or could we do these at the same time?

Reply:
https://msazure.visualstudio.com/One/_git/AzureUX-ARM/pullrequest/2210930?_a=overview PR created to change back end.